src/modules/api/transaction/services/index.ts

```diff
@@ -788,6 +789,131 @@ export class TransactionService {
         });
     }
 
+    async getUserTransactions(options: UserTransactionHistoryDto) {
+        const paginationMeta: Partial<PaginationMeta> = {};
+
+        const startDate = startOfMonth(new Date(options.date));
```

Comment:
why are you filtering by date? Is that what is in the figma?

Reply:
According to your explanation yesterday, when the filer button is clicked there'll be an option to search via status/date . Also I was asked to follow the template for get all transactions

Reply:
Is there a date filtring on the transaction history API?

Reply:
I've removed it